src/components/Select/Select.modules.scss

```diff
@@ -16,6 +16,8 @@
   &::-ms-expand {
     display: none;
   }
+
+  background-color: transparent;
```

Comment:
This ought to be an explicit colour, in our case `#ffffff` (or the equivalent SCSS variable) since it may render as grey on the AOSP Browser (Android 4.x) or other custom colours applied by users at the OS level.

Reply:
I think, I'm just going to remove it because it is not doing anything.
We already have `appearance: none;`.